package/MDAnalysis/core/universe.py

```diff
@@ -1459,13 +1491,108 @@ def from_smiles(cls, smiles, sanitize=True, addHs=True,
                 "hydrogens with `addHs=True`")
 
             numConfs = rdkit_kwargs.pop("numConfs", numConfs)
-            if not (type(numConfs) is int and numConfs > 0):
+            if not (isinstance(numConfs, int) and numConfs > 0):
                 raise SyntaxError("numConfs must be a non-zero positive "
-                "integer instead of {0}".format(numConfs))
+                                  "integer instead of {0}".format(numConfs))
             AllChem.EmbedMultipleConfs(mol, numConfs, **rdkit_kwargs)
 
         return cls(mol, **kwargs)
 
+    def guess_TopologyAttrs(
+            self, context=None, to_guess=(), force_guess=(), **kwargs):
+        """
+        Guess and add attributes through a specific context-aware guesser.
+
+        Parameters
+        ----------
+        context: str or :mod:`Guesser<MDAanalysis.guesser>` class
+            For calling a matching guesser class for this specific context
+        to_guess: list[str], (optional, default ``['types', 'masses']``)
+         .. warning::
```

Comment:
```suggestion

         .. warning::
```
From the looks of the render I _think_ this would lead to the intended behaviour?

![image](https://github.com/MDAnalysis/mdanalysis/assets/12460125/ed292e35-c919-4f12-ad16-3d8242594eb7)

Reply:
You need an additional empty line so that 

```
            TopologyAttrs to be guessed. These TopologyAttrs will be wholly
            guessed if they don't exist in the Universe. If they already exist
            in the Universe, only empty or missing values will be guessed.
```
does not show up _inside_ the warning text box.

FOllowing comments on a similar case above, perhaps rearrange so that the warning comes after the normal text
```
      to_guess: list[str], (optional, default ``['types', 'masses']``)
            TopologyAttrs to be guessed. These TopologyAttrs will be wholly
            guessed if they don't exist in the Universe. If they already exist
            in the Universe, only empty or missing values will be guessed.          

            .. warning:: 
                In MDAnalysis 2.x, types and masses are being automatically
                guessed if they are missing (``to_guess=('types, 'masses')``).
                However, starting with release 3.0 **no guessing will be done
                by default** and it will beup to the user to request guessing
                using ``to_guess`` and ``force_guess``.


      force_guess: list[str], (optional)
            TopologyAttrs in this list will be force guessed. If the
            ...
```

Note the indentation.